README.md

```diff
@@ -14,8 +17,10 @@ You are free to use whatever technologies you want, unless instructed otherwise.
 ## Procedure
 
 We would like you to send us (a link to) a git repository (that we can access).  
-We also need some documentation on how to run your app and how we can verify it will still run after we made some changes to it).  
```

Comment:
Not sure about the removal of "how we can verify it will still run after we made some changes to it" since that was the only reference to tests.

Reply:
The original sentence felt a bit weird though. An alternative anyone?